dashboard/app/models/peer_review.rb

```diff
@@ -30,14 +30,53 @@ class PeerReview < ActiveRecord::Base
   belongs_to :level
   belongs_to :level_source
 
+  after_save :mark_user_level
+
   REVIEWS_PER_SUBMISSION = 2
+  REVIEWS_FOR_CONSENSUS = 2
 
   enum status: {
```

Comment:
I should have mentioned this earlier, but convention with PLC objects has been to use strings as statuses instead of numbers. It makes for much easier debugging, and we have space to burn.